docs/content/1.guide/4.storage.md

```diff
@@ -110,46 +120,15 @@ export default defineNuxtConfig({
 ```
 ::
 
+::alert{type="warning"}
+This is a temporary workaround, with a better solution coming in the future! Keep a lookout on the GitHub issue [here](https://github.com/unjs/nitro/issues/1161#issuecomment-1511444675).
+::
 
-Usage:
-
-```js
-await useStorage('redis').setItem('foo', { hello: 'world' })
```

Comment:
Why removing examples?

Reply:
I refactor it and move it to the top (line 20).

Reply:
It was redondant with the beginning.